pwiz_tools/Skyline/TestConnected/ArdiaFunctionalTest.cs

```diff
@@ -203,9 +203,20 @@ protected override void DoTest()
             }
             
             WaitForDocumentLoaded();
-            RunUI(() => SkylineWindow.SelectElement(ElementRefs.FromObjectReference(ElementLocator.Parse("Molecule:/Molecules/Reserpine"))));
 
-            ClickChromatogram(0.5, 33000);
```

Comment:
If you swap the raw files above so caffeicquinic acid is first then you don't need to change this part, right?